assets/app/views/create/fromimage.html

```diff
@@ -0,0 +1,287 @@
+                               ng-disabled="!routing.customCerts|| routing.option === 'none'">
+                      </li>
+                      <li class="form-group">
+                        <label>Private Key</label>
+                        <textarea name="privatekey"
+                                  class="form-control"
+                                  ng-show="displaySecurityField('key')"
+                                  ng-model="routing.security.key" 
+                                  ng-disabled="!routing.customCerts|| routing.option === 'none'" 
+                                  ng-required="routing.customCerts"></textarea>
+                          <input type="file" 
+                                 ng-show="supportsFileUpload"
+                                 class="form-control"
+                                 onchange="angular.element(this).scope().uploadSecurityFile('key', this.files[0])" 
+                                 ng-disabled="!routing.customCerts|| routing.option === 'none'">
+                      </li>
+                      <li class="form-group">
+                        <label>CA Certificate</label>
+                        <textarea name="cacert"
+                                  class="form-control"
+                                  ng-show="displaySecurityField('caCertificate')"
+                                  ng-model="routing.security.caCertificate" 
+                                  ng-disabled="!routing.customCerts|| routing.option === 'none'" 
+                                  ng-required="routing.customCerts"></textarea>
+                          <input type="file"
+                                 class="form-control"
+                                  ng-show="supportsFileUpload"
+                                 onchange="angular.element(this).scope().uploadSecurityFile('caCertificate', this.files[0])" 
+                                 ng-disabled="!routing.customCerts|| routing.option === 'none'">
+                      </li>
+                    </ul>
+                  </li>
+                </ul>
+              </osc-form-section>
+              <osc-form-section 
+                header="Deployment Configuration" 
+                about-title="Deployments"
+                about="Deployment configurations describe how your application is configured
+                  by the cluster and under what conditions it should be recreated (e.g. when the image changes)."
+                >
+                <div ng-hide="$parent.expand">
+                  <div>
+                    <strong>Autodeploy when a new image is available: </strong>
+                    <span>{{deploymentConfig.deployOnNewImage | yesNo}}</span>
+                  </div>
+                  <div>
+                    <strong>Autodeploy when deployment configuration changes: </strong>
+                    <span>{{deploymentConfig.deployOnConfigChange | yesNo}}</span>
+                  </div>
```

Comment:
assuming this is ties to line 152, this line should not say "builder", it's just autodeploy when a new image is available.

what this should translate into is "when a new version of my application image is created, either because i manually pushed something, or a build pushed something, redeploy my app"

That's separate from what I think is still missing here which is "if my builder image (eg ruby-20-centos7) changed, rebuild my application".  That rebuild might then trigger a redeploy, but that's a separate issue.